internal/widget/scroller.go

```diff
@@ -489,7 +493,19 @@ func (s *Scroll) Resize(sz fyne.Size) {
 	}
 
 	s.Base.Resize(sz)
-	s.Refresh()
+	s.refreshBars()
+}
+
+// SetOffset will update the location of the content of this scroll container.
+//
+// Since: 2.6
+func (s *Scroll) SetOffset(p fyne.Position) {
```

Comment:
Should this perhaps be named `ScrollToOffset` to match the naming of `ScrollToBottom` etc, and also the `ScrollToOffset` that exists in List and GridWrap?

Reply:
Thanks, good eye.